src/crimson/os/futurized_store.cc

```diff
@@ -15,9 +17,14 @@ FuturizedStore::create(const std::string& type,
   } else if (type == "seastore") {
     return crimson::os::seastore::make_seastore(data, values);
   } else {
+#ifdef WITH_BLUESTORE
     // use AlienStore as a fallback. It adapts e.g. BlueStore.
     return std::make_unique<crimson::os::AlienStore>(
       type, data, values);
+#else
+    ceph_abort_msgf("unsupported objectstore type: %s", type.c_str());
```

Comment:
Restored the behavior we had before e6ed65db8b4e0a2f8026c2e35a12dd292c5f2b8c which introduced the regression.